From 88123162d16dbcdc41fc89f4affb23df41088ea8 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@st.com>
Date: Thu, 14 Apr 2022 17:33:05 +0200
Subject: [PATCH 167/954] feat(stm32mp2): enable watchdog driver

This patch enables watchdog support on STM32MP2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 9aa073ef09dcab24279f7a097a284325d4656a37 feat(stm32mp2): enable watchdog driver]
Signed-off-by: Christophe Roullier <christophe.roullier@st.com>
Change-Id: Ib953fc06498e9a68906c873600299ea87a5a2c61
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/249554
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Fabien DESSENNE <fabien.dessenne@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 plat/st/stm32mp2/platform.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plat/st/stm32mp2/platform.mk b/plat/st/stm32mp2/platform.mk
index 32683171a5..8eabd08207 100644
--- a/plat/st/stm32mp2/platform.mk
+++ b/plat/st/stm32mp2/platform.mk
@@ -137,6 +137,7 @@ PLAT_BL_COMMON_SOURCES		+=	plat/st/stm32mp2/${ARCH}/stm32mp2_helper.S
 PLAT_BL_COMMON_SOURCES		+=	plat/st/stm32mp2/stm32mp2_private.c
 
 PLAT_BL_COMMON_SOURCES		+=	drivers/st/bsec/bsec3.c					\
+					drivers/st/iwdg/stm32_iwdg.c				\
 					drivers/st/reset/stm32mp2_reset.c			\
 					plat/st/stm32mp2/stm32mp2_pwr.c				\
 					plat/st/stm32mp2/stm32mp2_syscfg.c
-- 
2.53.0

